openreview/profile/process/request_remove_name_decision_process.py

```diff
@@ -0,0 +1,222 @@
+            ## catch the error, some notes may not match with the invitation
+            try:
+                client.post_note(note)
+            except Exception as e:
+                print(f'note id {note.id} not updated: {e}')
+
+        signed_notes = client_v2.get_all_notes(signature=username)
+        for note in signed_notes:
+            signatures = []
+            for signature in note.signatures:
+                if username == signature:
+                    signatures.append(profile.id)
+                else:
+                    signatures.append(signature)
+
+            readers = []
+            for reader in note.readers:
+                if username == reader:
+                    readers.append(profile.id)
+                else:
+                    readers.append(reader)
+            writers = []
+            for writer in note.writers:
+                if username == writer:
+                    writers.append(profile.id)
+                else:
+                    writers.append(writer)
+            ## catch the error, some notes may not match with the invitation
+            try:
+                client_v2.post_note_edit(
+                    invitation = note.domain + '/-/Edit',
+                    readers = [note.domain],
+                    signatures = [SUPPORT_USER_ID],
+                    note = openreview.api.Note(
+                        id=note.id, 
+                        readers=readers,
+                        writers=writers,
+                        signatures=signatures
+                ))
+            except Exception as e:
+                print(f'note id {note.id} not updated: {e}')                
+
+        print('Rename all the edges')
+        head_edges = client.get_edges(head=username, limit=1)
+        tail_edges = client.get_edges(tail=username, limit=1)
+        if head_edges or tail_edges:
+            client.rename_edges(username, profile.id)
+        
+        print('Replace all the group members that contain the name to remove')
+        memberships = client.get_all_groups(member=username)
+
+        for group in memberships:
+            if username in group.members:
+                client.remove_members_from_group(group.id, username)
+                client.add_members_to_group(group.id, profile.id)
+
+
+        print('Post a profile reference to remove the name')
+        requested_name = {}
+        for name in profile.content['names']:
+            if username == name.get('username'):
+                requested_name = name
+        
+        group = client.get_group(username)
+        group.members = []
+        group.signatures = ['~Super_User1']
+        client.post_group(group)
+        
+        client.post_profile(openreview.Profile(
+            referent = profile.id, 
+            invitation = '~/-/invitation',
+            signatures = ['~Super_User1'],
+            content = {},
+            metaContent = {
+                'names': { 
```

Comment:
@carlosmondra this is the line where names is an obj

Reply:
I see thanks. I'm not sure if this is causing the issue. I'll try to reproduce it. It would be interesting to see if all the profiles that have requested name deletion have this issue.